documentation/source/roadmap.rst

```diff
@@ -5,6 +5,7 @@ Roadmap
 cocotb is in active development.
 
 We use GitHub issues to track our pending tasks.
-Take a look at the `open Feature List <https://github.com/cocotb/cocotb/issues?q=is%3Aissue+is%3Aopen+label%3Atype%3Afeature>`_ to see the work that's lined up.
+Take a look at the `open Feature List <https://github.com/cocotb/cocotb/issues?q=is%3Aissue+is%3Aopen+label%3Atype%3Afeature>`_ to see the work that's lined up,
```

Comment:
https://cocotb--1969.org.readthedocs.build/en/1969/roadmap.html

The captialization looks weird to me
```suggestion
Take a look at the `open feature list <https://github.com/cocotb/cocotb/issues?q=is%3Aissue+is%3Aopen+label%3Atype%3Afeature>`_ to see the work that's lined up,
```